apps/frontend/src/pages/projects/components/project-create-form/styles.module.css

```diff
@@ -0,0 +1,12 @@
+.form-wrapper {
+	display: flex;
+	flex-direction: column;
+	gap: 16px;
+	align-items: flex-end;
+}
+
+.button-wrapper {
+	display: flex;
+	flex-direction: column;
+	min-width: 150px;
```

Comment:
sounds like max-width will fit better here
```suggestion
	max-width: 150px;
```

Reply:
Button wrapper is placed inside flex container, that forces shrinking to min-content. So this should be `min-width` in order to prevent flex from shrinking the container to have width less than 150px (so flex shrinks it exact to 150px). If I use `max-width` instead, the width of the button (as the width of the wraper itself) will be less than 150px;
![image](https://github.com/user-attachments/assets/36d5cc09-1372-4756-af0d-a638c5fb6bb6)
↑ with `max-width: 150px`
![image](https://github.com/user-attachments/assets/34f67fff-fc01-4f79-b9c3-5eef00db5ca6)
↑ with `min-width: 150px`

Reply:
So I left `min-width`